src/RA_Dlg_MemBookmark.cpp

```diff
@@ -695,82 +700,66 @@ void Dlg_MemBookmark::ExportJSON()
 
     if (m_vBookmarks.size() == 0)
     {
-        MessageBox(nullptr, _T("No bookmarks to save: please create a bookmark before attempting to save."), _T("Error!"), MB_OK);
+        MessageBox(nullptr, _T("No bookmarks to save: please create a bookmark before attempting to save."),
+                   _T("Error!"), MB_OK);
         return;
     }
 
-    std::string defaultDir = g_sHomeDir + RA_DIR_BOOKMARKS;
-
-    CComPtr<IFileSaveDialog> pDlg;
-
-    HRESULT hr;
-    if (SUCCEEDED(hr = CoCreateInstance(CLSID_FileSaveDialog, nullptr, CLSCTX_ALL, IID_IFileSaveDialog, reinterpret_cast<void**>(&pDlg))))
```

Comment:
Similar comment here. I suspect this wasn't working on XP before: https://github.com/RetroAchievements/RAIntegration/blob/610021460e6a1d4aa42f5892cd31240f7cde5e6d/src/RA_Dlg_MemBookmark.cpp#L710